### PR TITLE
Ch839/topic ui refinement

### PR DESCRIPTION
**Pull request by kim-lindhard-dfds**

### CODE REVIEW

**Comment by kim-lindhard-dfds** [comment]
[//]: # (4744e426-5795-11e9-8647-d663bd873d93)

|| **CodeScene Delta Analysis Results** |
| --- | ---- |
| **Risk** |5|
| **Quality Gates** |OK|
| **Description** |The change is medium risk as it is more diffused (4 files modifified, 22 lines added, 13 lines deleted) than your typical change set. The risk is lower since it is a very experienced author with many contributions.|
| **Improvements** |TestCapabilityApiController.cs improves its Code Health from 9 -> 9.33|

File: src/Blaster.WebApi/Features/Shared/userservice.js [hunk 1]
```diff
@@ -1,134 +1,134 @@
...
+
+export default class UserService {
+    constructor(configuration)
+    {
+        if (!configuration) {
+            configuration = {
+                auth: {
+                    clientId: "91c38c20-4d2c-485d-80ac-a053619a02db",
+                    authority: "https://login.microsoftonline.com/73a99466-ad05-4221-9f90-e7142aa2f6c1",
+                    redirectUri: "http://localhost:4200"
```
Comment (SEQUOIIA) [inline src/Blaster.WebApi/Features/Shared/userservice.js]: Nope.

**Comment by kim-lindhard-dfds** [comment]
[//]: # (4744e426-5795-11e9-8647-d663bd873d93)

|| **CodeScene Delta Analysis Results** |
| --- | ---- |
| **Risk** |5|
| **Quality Gates** |OK|
| **Description** |The change is medium risk as it is more diffused (4 files modifified, 23 lines added, 14 lines deleted) than your typical change set. The risk is lower since it is a very experienced author with many contributions.|
| **Improvements** |TestCapabilityApiController.cs improves its Code Health from 9 -> 9.33|

**Comment by kim-lindhard-dfds** [comment]
[//]: # (4744e426-5795-11e9-8647-d663bd873d93)

|| **CodeScene Delta Analysis Results** |
| --- | ---- |
| **Risk** |5|
| **Quality Gates** |OK|
| **Description** |The change is medium risk as it is more diffused (4 files modifified, 24 lines added, 15 lines deleted) than your typical change set. The risk is lower since it is a very experienced author with many contributions.|
| **Improvements** |TestCapabilityApiController.cs improves its Code Health from 9 -> 9.33|

**Pull request merged**